content/en/docs/v3.6/install.md

```diff
@@ -79,7 +79,7 @@ TBD---Help Wanted
 
 ## Installation on Kubernetes, using a statefulset or helm chart
 
-TBD---Help Wanted
+You can follow the instructions provided by [Bitnami's etcd Helm chart]
```

Comment:
As above, think we should just say why we are directing the user towards the Bitnami chart.